extensions/amp-video-iframe/amp-video-iframe.md

```diff
@@ -424,6 +424,62 @@ custom variables to log. These are available as `VIDEO_STATE`, keyed by name
 prefixed with `custom_`, i.e. the object `{myVar: 'foo'}` will be available as
 `{'custom_myVar': 'foo}`.
 
+#### <a name="getConsent"></a> `getConsent(callback)`
+
+The iframe document can request user consent data when the host document uses
+[`amp-consent`](hhttps://amp.dev/documentation/components/amp-consent/).
+
+Note: If you only require to block the iframe from loading when consent is not given, it's preferable to [set the `data-block-on-consent` attribute](https://amp.dev/documentation/components/amp-consent/#basic-blocking-behaviors) instead of calling `getConsentData()`
+
+The `callback` passed to the function will be executed with an object that looks
+like this:
+
+```json
+{
+  "consentMetadata": {
+    "consentStringType": 2,
+    "additionalConsent": "additional-consent-string",
+    "gdprApplies": true,
+    "purposeOne": true
+  },
+  "consentString": "accept-string",
+  "consentPolicyState": 1
+}
+```
```

Comment:
@micajuine-ho It would be great to describe what these keys are, but I don't have that information. Do we document `consentMetadata`, `consentString` and `consentPolicyState` anywhere?

Reply:
We document them here: https://github.com/ampproject/amphtml/blob/master/extensions/amp-consent/customizing-extension-behaviors-on-consent.md#advanced-blocking-behaviors